docs/DeveloperGuide.md

```diff
@@ -543,11 +544,11 @@ In the following section, the interaction between different objects will be disc
 
    After receiving an input from user for edit command, `EditCommandParser#parse` will be invoked.
 
-   As mentioned in above section, if either one of the compulsory prefixes are missing, `ParseException` will be thrown to remind users. Furthermore, invalid values supplied for `type` and `index` (e.g. `i/test`), a `ParseException` will be thrown.
+   As mentioned in above section, if either one of the compulsory prefixes is missing, `ParseException` will be thrown to remind users. Furthermore, invalid values supplied for `type` and `index` (e.g. `String` value for `index`), a `ParseException` will be thrown.
```

Comment:
I think this should be i/test instead, since we mentioned that String may not be clear :)

Reply:
Okay I think it's because I didn't check the content carefully, thanks!